### PR TITLE
Hydration refactor

### PR DESCRIPTION
**Pull request by percworld**
### Description

* This refactor is based on the data model update
*  There are no arguments passed to methods, as the Hydration instance has a us

* Start at line 8 of Hydration.js

*These changes were motivated by the last pull request and the desire of the team to merge both classes ( with Activity)

*** needs testing methods in test files

### CODE REVIEW

File: src/User-repo.js [hunk 1]
```diff
@@ -2,14 +2,15 @@ import User from "./User";
 
 class UserRepo {
   constructor(users) {
-    this.users = users
-  };
+    this.users = users;
+    this.stepGoals = this.users.map(user => user.stepGoal)/this.users.length;
```
Comment (PaigeVannelli) [inline src/User-repo.js:6]: We should pull this over to the activity class

File: src/scripts.js [hunk 2]
```diff
@@ -50,33 +50,41 @@ var bestUserSteps = document.getElementById('bestUserSteps');
...
-    let hydrationObject = new Hydration(hydrationData, user);
+
+  function displayHydrationData(hydrationData, user, userRepo) {
+    let hydrationObject = new Hydration(hydrationData, user, today, userRepo);
     let averageHydration = hydrationObject.calculateAverageOunces();
     hydrationAverage.insertAdjacentHTML('afterBegin', `<p>Your average water intake is</p><p><span class="number">${averageHydration}</span></p> <p>oz per day.</p>`);
+    hydrationToday.insertAdjacentHTML('afterBegin', `<p>You drank</p><p><span class="number">${hydrationObject.calculateDailyOunces()}</span></p><p>oz water today.</p>`); //userRepo.getToday()
+    const weekHydrationRecord = hydrationObject.hydrationData.filter(drink => drink.userID === hydrationObject.user.id);
+    hydrationThisWeek.insertAdjacentHTML('afterBegin', makeHydrationHTML(hydrationObject.user.id, hydrationObject, hydrationObject.user, hydrationObject.calculateFirstWeekOunces()));
+    hydrationEarlierWeek.insertAdjacentHTML('afterBegin', makeHydrationHTML(hydrationObject.user.id, hydrationObject, hydrationObject.user, hydrationObject.calculateRandomWeekOunces()));
```
Comment (PaigeVannelli) [inline src/scripts.js:83]: Need to pass into charts to display

**Review by PaigeVannelli — APPROVED** [review]
Looks good!

**Comment by josharagon** [comment]
Changes look great... I will fix my circle chart to be dynamic and fit these new hydration methods!

**Pull request merged**